src/Dispatcher.h

```diff
@@ -167,6 +167,9 @@ class Dispatcher {
   Packet* obtainNewPacket();
   void releasePacket(Packet* packet);
   void sendPacket(Packet* packet, uint8_t priority, uint32_t delay_millis=0);
+  uint32_t getTotalOutboundCount() {
```

Comment:
This is exposing more data than the API client currently needs. Could it simplify to `bool hasOutboundPackets() { ... }`

That simplifies the callsite to:

```c++
if (the_mesh.getIdleInterval() &&
      ((millis() - active_timestamp) > active_time_ms) &&
      the_mesh.hasOutboundPacket()) {
    radio_driver.blockTaskUntilRXEvent(the_mesh.getIdleInterval() * 1000);
    ...
  }
```

It's easier to reason about negating a `bool` than negating a `uint32_t`.

Reply:
Good idea, yes.